### DISCUSSION TITLE
第5节课作业(20班)

### DISCUSSION THREAD

**Discussion by Farewell-CK** [InternLM/Tutorial#459] [作业]
第五节课作业，20班的宝子们

**Comment by liu673** [comment]
https://github.com/liu673/InternLM_OpenNotebook/blob/master/lesson_5/lesson_5_homework.md

**Comment by deyouzuishuai** [comment]
https://zhuanlan.zhihu.com/p/679626590

**Comment by 20300240011** [comment]
https://zhuanlan.zhihu.com/p/679656063

**Comment by LiyanJin** [comment]
https://blog.csdn.net/weixin_53443275/article/details/135873703
